### ISSUE TITLE
Test OSX .app file

### ISSUE THREAD

**Issue by ethanwhite** [weecology/retriever#141]
I'm in the process of trying to get the EcoData Retriever fully functional on OSX (since so many of the awesome ecological informaticsy people I know use Macs). As I've mentioned elsewhere it looks like building from source now works, at least when using homebrew (http://ecodataretriever.org/getting_started.html).

What I'm working on now is getting the .app working so that you don't need to be comfortable in the shell (and have XCode installed) to use the Retriever. I have a version that is working on the machine that I built it on (our only Mac) and was wondering if some kind Mac folks like @karthik, @sckott, @emhart, @sarahsupp and @dfalster might have a few minutes to give it a trial run.

The file is here: https://www.dropbox.com/s/26b1pj91mqucc0l/retriever.zip

Basically I'm just looking for folks to unzip it, double click on it, and see if:
1. It opens at all. 
2. You can install things successfully when setting the database management system to CSV and sqlite (these don't have any external dependencies).
3. If you have either MySQL or PostgreSQL installed if it works with them. (MySQL is a bit fragile at the moment. It is currently working for most datasets, but not all, so just try a few if you get errors).
4. Report back.

Thanks in advance. And, yes, I wrote this issue... on a Mac.
